src/hooks/useFormState.js

```diff
@@ -29,6 +29,10 @@ export function useFormState(initialState) {
     }
   }
 
+  function getFormInteractedWith() {
+    return touched
```

Comment:
I double checked `setStateFactory` which is where this is set and basically the first time a button select group is clicked, or, a character is entered into a field (text, birthdate, zip) `touched` (for the form) becomes true. So, this works for our purposes.

I was thinking of exposing `getFieldValues` but Nick pointed out that was meant to be a private variable, and also I was thinking if we somehow checked all field values are `null` to determine if the form was interacted with, that could get pretty inefficient with a form that has a lot of fields.

Happy that we already had a flag to check and expose this.